api/activetigger/models.py

```diff
@@ -695,14 +702,19 @@ def add(self, element):
         """
         if element["status"] == "training":
             # update bdd status
-            self.db_manager.change_model_status(self.project_slug, element["model"].name, "trained")
+            self.db_manager.change_model_status(
```

Comment:
Do you not need to use projects_service here ? There is no more self.db_manager no ? (and I am not sure : since we can use different submodule of db_manager, don't you thing we rather need to keep self.db_manager as an attribute rather to reduce it at db_manager.projects_service

Reply:
It is the only question I have @JbIPS, will merge after that. Thanks

Reply:
Well catch, you're right. I also noticed the line is duplicated, l.705 is a residue of a failed merge attempt, l.708 is the right one.

I'll clean that immediatly